src/components/DocWrapper/DocList/DocList.test.tsx

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable react/jsx-props-no-spreading */
```

Comment:
You can remove this. Take a pull from development. Test files have been allowed props spreading.